### ISSUE TITLE
[nwnx_optimizations] GetObjectTag lookup fails

### ISSUE THREAD

**Issue by LiarisFreeze** [nwnxee/unified#842]
Bug. 
I use the nwnx_optimizations plugin. 
Area is created and created point in it (create new unique tag point). 
GetObjectTag does not find point.

Check, I have it like that.
When I turn off the plugin - everything is fine.

**Comment by plenarius** [comment]
No longer relevant after #1103
